haystack/nodes/prompt/invocation_layer/hugging_face.py

```diff
@@ -119,8 +119,15 @@ def __init__(
 
         if len(model_input_kwargs) > 0:
             logger.info("Using model input kwargs %s in %s", model_input_kwargs, self.__class__.__name__)
-        self.task_name = get_task(model_name_or_path, use_auth_token=use_auth_token)
+
+        # If task_name is not provided, get the task name from the model name or path (uses HFApi)
+        if "task_name" in kwargs:
```

Comment:
Can you add `task_name` to the list of accepted parameters in the docstring above, line 50?

Reply:
Yes